SimulationAgent/Program.cs

```diff
@@ -26,7 +26,13 @@ private static void PrintBootstrapInfo(IContainer container)
             var config = container.Resolve<IConfig>();
             logger.Info("Simulation agent started", () => new { Uptime.ProcessId });
             logger.Info("Device Models folder: " + config.ServicesConfig.DeviceModelsFolder, () => { });
-            logger.Info("Scripts folder:      " + config.ServicesConfig.DeviceModelsScriptsFolder, () => { });
+            logger.Info("Scripts folder:       " + config.ServicesConfig.DeviceModelsScriptsFolder, () => { });
+
+            logger.Info("Connections per sec:  " + config.ServicesConfig.RateLimiting.ConnectionsPerSecond, () => { });
```

Comment:
should we wrap these under config entries for S1/S2/S3 with 1 Unit?

Reply:
I thought about having configurations per SKU, but I think that's now how people will consume the service. I added the comments in application.ini to explain how to calculate values, but I prefer the fully customizable values, that don't have to match exactly the SKUs. For example, I might want to reduce the number of twin ops more than other limits, because of Web UI (or other UIs)

Reply:
Got it - this makes sense WRT customizing for the # of clients.  If you have a low # of people on the cloud side accessing the data I might configure simulation to run closer to the hub limits.